src/client.ts

```diff
@@ -1460,20 +1460,23 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
    * @param {BannedUsersFilters} filterConditions MongoDB style filter conditions
    * @param {BannedUsersSort} sort Sort options [{created_at: 1}].
    * @param {BannedUsersPaginationOptions} options Option object, {limit: 10, offset:0}
+   * @param {boolean} excludeExpiredBans
    *
    * @return {Promise<BannedUsersResponse<StreamChatGenerics>>} Ban Query Response
    */
   async queryBannedUsers(
     filterConditions: BannedUsersFilters = {},
     sort: BannedUsersSort = [],
     options: BannedUsersPaginationOptions = {},
+    excludeExpiredBans = false,
```

Comment:
don't add it explicitely. Just add `excludeExpiredBans` to type `BannedUsersPaginationOptions`